frontend/src/components/layout/Header.tsx

```diff
@@ -135,7 +154,7 @@ const Header = () => {
                             onChange={onChange}
                             value={query}
                             className="block w-full rounded-full bg-bgColorPrimary py-2 pl-10 pr-3 text-sm placeholder-gray-500 focus:border-textPrimary focus:placeholder-gray-400 focus:outline-none focus:ring-1 focus:ring-textPrimary dark:text-textPrimary dark:focus:text-white sm:text-sm"
-                            placeholder={t("search-apps")}
+                            placeholder={'/ '+t("search-apps")}
```

Comment:
I'm wondering, if we can make this more obvious, via an icon or something.

Is there any prior art, that already has a function like this? Most desktop apps seem to do this on `Control + K` for e.g.

Reply:
The two most common shortcuts are ctrl k and /.

Google, github and MDN use /, the Tailwind documentation uses ctrl + k.

Personally, I am not a big fan of Ctrl k because it interferes with the browser's default shortcuts.

Reply:
Hrm, didn't know google, github and mdn have this. I find googles implementation quite strange. Githubs seems to be the one that communicates it best.

Only thing I'm slightly worried about is, if this causes problems when you enter `/` into other textboxes on the page.